src/config/dynamic-theme-fixes.config

```diff
@@ -5257,6 +5257,30 @@ INVERT
 
 ================================
 
+circuits.vcn.com
+
+INVERT
+img
```

Comment:
Please try being more specific (refer elements by class), as far as every image on every page will be inverted.

Reply:
Gotcha. I added an attribute specifier to only invert images that are sourced from the graphing backend.

Reply:
In case you're wondering why I chose referencing them by src tags that start with "/rrd_graph.php", it's because there are no class attributes that I can see for these types of images. The image tag is just `<img src="/rrd_graph.php?size=550x200&service_id=43916&ds=ifHCInOctets:ifHCOutOctets&timeline=daily" border="0" style="padding-top:4px">` or similar.